### DISCUSSION TITLE
Why do you require a phone number to use your Discord Server ?

### DISCUSSION THREAD

**Discussion by cassepipe** [TanStack/query#5312] [General] [answered]
Tanstack seems like a great project but phonewalling the commnunity is a bit sad.

I don't want to give you or Discord my phone...

Please reconsider 🙏

**Comment by TkDodo** [comment]
I don't know anything about that, sorry. Is that a setting we can change? Is that a discord requirement?

**Reply by cassepipe** [reply]
Thanks for reaching out. I have not set up Discord servers myself so I wouldn't know. I doubt this is the default but I will look into it.

**Reply by TkDodo** [reply]
confirmed with the mods that this is turned on to avoid spam bots signing up.

**Reply by cassepipe** [reply]
All the other servers I participate in seem to be doing fine without it....
Maybe a requirement could be instead that one has to introduce itself rather in the #introductions channel before being able to participate ?
Anyways, I guess this is the mod's call in the end since they are doing the work.
Oh well, I'll participate to the community here instead.

**Comment by louis-young** [comment]
I don't _think_ a phone number is required to use the Discord server per se, I think it's used as a form of verification used to prevent people from spamming certain channels (the core package chat channels if I remember correctly e.g. query, table, virtual and router). Please can you let me know if this is correct? I think you should have access to the question channels without verifying your phone number. Thanks 🙂

**Reply by cassepipe** [reply]
Thanks for caring ! 
I just got into the server and I can't participate in *any* channel wheter a it is a regular Channel a "Post" one.
Not even the #introductions channel :)

**Reply by cassepipe** [reply]
Oh I think I misunderstood your initial message, you're saying they're asking for *any* phone number at signup for verfication purposes. Still, it's compulsory for *all* channels

Even if it's not meant to be kept around I don't trust any commercial entity (like Discord) not to store that data somewhere. I am not paranoid but I don't want my phone number/adress to be stored on the internet by any service there is if I can help it. Maybe it's a lost battle but I am not giving up unless I don't have any other choice.

**Comment by TkDodo** [comment, marked as answer]
I checked once more, and the answer is:

> Without it, we were getting massive amounts of spam for web3, scams and porn including DM abuse.

So, it's a safety feature. Sorry.
